api/aggregators/searchAggregator.js

```diff
@@ -121,7 +137,7 @@ exports.createSortingPagingAggr = function(schemaName, sortValues, sortField, so
 
   // We don't want to have sort in the aggregation if the front end doesn't need sort.
   if (sortField && sortDirection) {
-    if (defaultTwoSorts){
+     if(dateHandling){
```

Comment:
Sort of implies this is a handler for any sorted date field, even though it's specific to sorting with datePosted. MIght want to rename for clarity